tic-tac-toe-app/App.jsx

```diff
@@ -1,5 +1,4 @@
-const App = () => {
-	return <div>A tic-tac-toe game</div>;
-};
+import React from 'react';
+import Game from './components/Game'
```

Comment:
Importing React and the components into the file is standard. Though, due to the scripts used within index.js, these imports aren't necessary.